var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -72,7 +72,11 @@ class Kokkos(Package):
                   'Pascal60', 'Pascal61',
                   'Volta70', 'Volta72')
 
-    cuda_options = ('force_uvm', 'use_ldg', 'rdc', 'enable_lambda')
+    # C++ standard variant
+    variant('cxxstandard', default='none',
```

Comment:
Can you please change the variant name to `cxxstd`? It would improve naming consistency since `cxxstd` is already used in 16 packages.